packages/components/src/ui/radix-dropdown/index.tsx

```diff
@@ -0,0 +1,255 @@
+export const DropdownMenu = ( {
+	// Root props
+	defaultOpen,
+	open,
+	onOpenChange,
+	modal = true,
+	// Content positioning props
+	side = 'bottom',
+	sideOffset = 0,
+	align = 'center',
+	alignOffset = 0,
+	// Render props
+	children,
+	trigger,
+}: DropdownMenuProps ) => {
+	return (
+		<DropdownMenuPrimitive.Root
+			defaultOpen={ defaultOpen }
+			open={ open }
+			onOpenChange={ onOpenChange }
+			modal={ modal }
+		>
+			<DropdownMenuPrimitive.Trigger asChild>
+				{ trigger }
+			</DropdownMenuPrimitive.Trigger>
+			<DropdownMenuPrimitive.Portal>
+				<DropdownMenuStyled.Content
+					side={ side }
+					align={ align }
+					sideOffset={ sideOffset }
+					alignOffset={ alignOffset }
+					loop={ true }
+				>
+					{ children }
+				</DropdownMenuStyled.Content>
+			</DropdownMenuPrimitive.Portal>
+		</DropdownMenuPrimitive.Root>
+	);
+};
+
+export const DropdownSubMenuTrigger = ( {
+	prefix,
+	suffix = <Icon icon={ chevronRightSmall } size={ 28 } />,
+	children,
+}: DropdownSubMenuTriggerProps ) => {
+	return (
+		<>
+			{ prefix && (
+				<DropdownMenuStyled.ItemPrefixWrapper>
+					{ prefix }
+				</DropdownMenuStyled.ItemPrefixWrapper>
+			) }
+			{ children }
+			{ suffix && (
+				<DropdownMenuStyled.ItemSuffixWrapper>
+					{ suffix }
+				</DropdownMenuStyled.ItemSuffixWrapper>
+			) }
+		</>
+	);
+};
+
+export const DropdownSubMenu = ( {
+	// Sub props
+	defaultOpen,
+	open,
+	onOpenChange,
+	// Sub trigger props
+	disabled,
+	textValue,
+	// Render props
+	children,
+	trigger,
+}: DropdownSubMenuProps ) => {
+	return (
+		<DropdownMenuPrimitive.Sub
+			defaultOpen={ defaultOpen }
+			open={ open }
+			onOpenChange={ onOpenChange }
+		>
+			<DropdownMenuStyled.SubTrigger
+				disabled={ disabled }
+				textValue={ textValue }
+			>
+				{ trigger }
+			</DropdownMenuStyled.SubTrigger>
+			<DropdownMenuPrimitive.Portal>
+				<DropdownMenuStyled.SubContent
+					loop
+					sideOffset={ 4 }
+					alignOffset={ -8 }
+				>
+					{ children }
+				</DropdownMenuStyled.SubContent>
+			</DropdownMenuPrimitive.Portal>
+		</DropdownMenuPrimitive.Sub>
+	);
+};
+
+export const DropdownMenuLabel = ( props: DropdownMenuLabelProps ) => (
+	<DropdownMenuStyled.Label { ...props } />
+);
+
+export const DropdownMenuGroup = ( props: DropdownMenuGroupProps ) => (
+	<DropdownMenuPrimitive.Group { ...props } />
+);
+
+export const DropdownMenuItem = forwardRef(
+	(
+		{ children, prefix, suffix, ...props }: DropdownItemProps,
+		forwardedRef: React.ForwardedRef< any >
+	) => {
+		return (
+			<DropdownMenuStyled.Item { ...props } ref={ forwardedRef }>
+				{ prefix && (
+					<DropdownMenuStyled.ItemPrefixWrapper>
+						{ prefix }
+					</DropdownMenuStyled.ItemPrefixWrapper>
+				) }
+				{ children }
+				{ suffix && (
+					<DropdownMenuStyled.ItemSuffixWrapper>
+						{ suffix }
+					</DropdownMenuStyled.ItemSuffixWrapper>
+				) }
+			</DropdownMenuStyled.Item>
+		);
+	}
+);
+
+export const DropdownMenuCheckboxItem = ( {
```

Comment:
Noting that checkbox and radio items don't have a prefix, since the prefix slot is taken by the check/radio icons